src/app/pages/LiquidityMining/components/MiningPool/PoolChart.tsx

```diff
@@ -1,312 +1,123 @@
-    borrow_apr: '655796008',
-    timestamp: '2021-05-14T00:00:04.000Z',
-  },
-  {
-    supply_apr: '3524504329',
-    borrow_apr: '655796008',
-    timestamp: '2021-05-14T06:00:01.000Z',
-  },
-  {
-    supply_apr: '3524504329',
-    borrow_apr: '655796008',
-    timestamp: '2021-05-14T12:00:02.000Z',
-  },
-  {
-    supply_apr: '3524504329',
-    borrow_apr: '655796008',
-    timestamp: '2021-05-14T18:00:01.000Z',
-  },
-];
+export function PoolChart(props: Props) {
+  const { t } = useTranslation();
+  const [btcHistory, setBtcHistory] = useState<ChartData>([]);
+  const [assetHistory, setAssetHistory] = useState<ChartData>([]);
+  const [totalHistory, setTotalHistory] = useState<ChartData>([]);
 
-const testTotal: any[] = [
-  {
-    total: '136.2222',
-    timestamp: '2021-05-08T00:00:02.000Z',
-  },
-  {
-    total: '146.1727',
-    timestamp: '2021-05-08T06:00:01.000Z',
-  },
-  {
-    total: '147.1727',
-    timestamp: '2021-05-08T12:00:01.000Z',
-  },
-  {
-    total: '143.1727',
-    timestamp: '2021-05-08T18:00:01.000Z',
-  },
-  {
-    total: '145.18',
-    timestamp: '2021-05-09T00:00:01.000Z',
-  },
-  {
-    total: '146.2222',
-    timestamp: '2021-05-09T06:00:03.000Z',
-  },
-  {
-    total: '148.5',
-    timestamp: '2021-05-09T12:00:02.000Z',
-  },
-  {
-    total: '149.78',
-    timestamp: '2021-05-09T18:00:02.000Z',
-  },
-  {
-    total: '152.555555',
-    timestamp: '2021-05-10T00:00:01.000Z',
-  },
-  {
-    total: '151.1727',
-    timestamp: '2021-05-10T06:00:03.000Z',
-  },
-  {
-    total: '153.1766',
-    timestamp: '2021-05-10T12:00:01.000Z',
-  },
-  {
-    total: '154.44',
-    timestamp: '2021-05-10T18:00:01.000Z',
-  },
-  {
-    total: '154.7',
-    timestamp: '2021-05-11T00:00:01.000Z',
-  },
-  {
-    total: '155.8888',
-    timestamp: '2021-05-11T06:00:01.000Z',
-  },
-  {
-    total: '163.1727',
-    timestamp: '2021-05-11T12:00:01.000Z',
-  },
-  {
-    total: '165.1727',
-    timestamp: '2021-05-11T18:00:01.000Z',
-  },
-  {
-    total: '166.1727',
-    timestamp: '2021-05-12T00:00:01.000Z',
-  },
-  {
-    total: '166.1727',
-    timestamp: '2021-05-12T06:00:02.000Z',
-  },
-  {
-    total: '166.1727',
-    timestamp: '2021-05-12T12:00:00.000Z',
-  },
-  {
-    total: '168.1727',
-    timestamp: '2021-05-12T18:00:05.000Z',
-  },
-  {
-    total: '167.1727',
-    timestamp: '2021-05-13T00:00:03.000Z',
-  },
-  {
-    total: '166.1727',
-    timestamp: '2021-05-13T06:00:02.000Z',
-  },
-  {
-    total: '167.727',
-    timestamp: '2021-05-13T12:00:03.000Z',
-  },
-  {
-    total: '168.1727',
-    timestamp: '2021-05-13T18:00:01.000Z',
-  },
-  {
-    total: '167.1727',
-    timestamp: '2021-05-14T00:00:04.000Z',
-  },
-  {
-    total: '167.666',
-    timestamp: '2021-05-14T06:00:01.000Z',
-  },
-  {
-    total: '169.1727',
-    timestamp: '2021-05-14T12:00:02.000Z',
-  },
-  {
-    total: '170.1727',
-    timestamp: '2021-05-14T18:00:01.000Z',
-  },
-];
+  useEffect(() => {
+    if (
+      props.history &&
```

Comment:
I would destructure the props so you don't have to prefix them and you can get rid of this line and use `history?.data && history?.balanceHistory && pool`.